app/institutions/dashboard/-components/chart-kpi-wrapper/chart-kpi/component.ts

```diff
@@ -0,0 +1,112 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { ChartData, ChartOptions } from 'ember-cli-chart';
+import Intl from 'ember-intl/services/intl';
+// eslint-disable-next-line max-len
+import { ChartDataModel, KpiChartModel } from 'ember-osf-web/institutions/dashboard/-components/chart-kpi-wrapper/component';
+
+interface KPIChartWrapperArgs {
+    data: KpiChartModel;
+}
+
+interface DataModel {
+    name: string;
+    total: number;
+    color: string;
+}
+
+export default class ChartKpi extends Component<KPIChartWrapperArgs> {
+    @service intl!: Intl;
+
+    @tracked expanded = false;
+    @tracked expandedData = [] as DataModel[];
+
+    /**
+     * chartOptions
+     *
+     * @description A getter for the chartjs options
+     *
+     * @returns a ChartOptions model which is custom to COS
+     */
+    get chartOptions(): ChartOptions {
+        return {
+            aspectRatio: 1,
+            legend: {
+                display: false,
+            },
+            scales: {
+                xAxes: [{
+                    display: false,
+                }],
+                yAxes: [{
+                    display: false,
+                }],
+            },
+        };
+    }
+
+    /**
+     * getColor
+     *
+     * @description Gets a specific color using a modulus
+     *
+     * @param index The index to retrieve
+     *
+     * @returns the color
+     */
+    private getColor(index: number): string {
+        const backgroundColors = [
+            '#00D1FF',
+            '#009CEF',
+            '#0063EF',
+            '#00568D',
+            '#004673',
+            '#00375A',
+            '#263947',
+        ];
+
+        return backgroundColors[index % backgroundColors.length];
+    }
+
+    /**
+     * chartData
+     *
+     * @description Transforms the standard chart data into data the charts can display
+     *
+     * @returns void
+     */
+    get chartData(): ChartData {
+        const backgroundColors = [] as string[];
+        const data = [] as number[];
+        const labels =  [] as string[];
+
+        this.args.data.chartData.map((chartData: ChartDataModel, $index: number) => {
```

Comment:
Very minor nit, but I went down a small rabbithole of Array.forEach vs Array.map and according to MDN
> Since map builds a new array, calling it without using the returned array is an anti-pattern; use [forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) or [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) instead.

It seems we aren't using the new newly built array from this `map`, so would it be reasonable to change this to `forEach`?